src/less/mixins/private/selection-list-mixins.less

```diff
@@ -3,8 +3,9 @@
     border-style: solid;
     border-width: 1px;
     box-sizing: border-box;
-    display: inline-flex;
+    display: inline-grid;
```

Comment:
Can we simply use `inline-block` here? Do we really need a `grid` display? On the surface, this layout seems like a simple stacked design. Are there any other views that we need to shuffle things around for?

Reply:
The problem is we have the following layout
```
<main value>                    <checkbox>
<subline>
```
The main thing is we dont want this to break other layouts
If you have any suggestions on how to do that, I'd be happy to see. The problem is we also want to limit the main value to 1 line and do ellipsis if its too long.

Reply:
Oh, my bad. I didn't realize there was a checkbox too. No, what you have is perfect, in that case.

Reply:
I know there aren't any designs with this yet, but there has been discussion of this format as well (which would not work with the grid approach):
```
<icon>      <main value>      <checkbox>
            <subline>
```

Reply:
Actually it would. That's exactly what we have for page notice. It would offer more flexibility. 
The issue is making sure we're not breaking existing layouts. This is the safest for that and means we dont have two layouts for subtitle or not.

Reply:
I like grid, In Phone input, we have, 
`<icon> <main value> <subtext>`